connector/launchers/runtime-embedded/build.gradle.kts

```diff
@@ -19,6 +19,10 @@ dependencies {
         // this will remove the RemoteDataPlaneSelectorService
         exclude(group = "org.eclipse.edc", "data-plane-selector-client")
     }
+    runtimeOnly(project(":launchers:api")) {
+        // this will remove the RemoteDataPlaneSelectorService
+        exclude(group = "org.eclipse.edc", "data-plane-selector-client")
```

Comment:
the `:launchers:api` does not define the `data-plane-selector-client` extension. no need to exclude it